test/fixtures/bpmn/non-regression/markers-fixed-size.bpmn

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
:warning: the tests are failing because the BPMN diagram contains labels and no diff threshold is defined, I suggest to
- remove the labels as we are not testing them here, this will make tests pass on ubuntu and macOS (the reference image seems to have been created on windows :smile: )
- rename this file and the other marker diagram to have a counter in the name like this is done for labels and pools (for instance, `labels.01.general.bpmn` and `labels.02.position-and-line-breaks.bpmn`). This helps keeping the files/tests grouped